funcs.php

```diff
@@ -164,7 +164,7 @@ function branches(
     if (isset(DO_NOT_MERGE_UP_FROM_MAJOR[$githubRepository])) {
         $doNotMergeUpFromMajor = DO_NOT_MERGE_UP_FROM_MAJOR[$githubRepository];
         $branches = array_filter($branches, function($branch) use ($doNotMergeUpFromMajor) {
-            return version_compare($branch, $doNotMergeUpFromMajor, '>');
+            return version_compare($branch, "$doNotMergeUpFromMajor.0", '>');
```

Comment:
I think we need to be a bit more sophisticated than this - we need to handle `3`, `3.0`, and any theoretical `3.x` branches, since this is intended to be a general solution we can reuse in the future, not just for linkfield.

Reply:
Unless I've misunderstood - in which case please add a test for `3.3` or something to validate that it will handle all those as well.

Reply:
Good spot, yes would have broken on 3.1.  Have updated and added tests.